package.json

```diff
@@ -69,6 +69,7 @@
     "@material/card": "5.1.0",
     "@material/checkbox": "5.1.0",
     "@material/chips": "5.1.0",
+    "@material/circular-progress": "6.0.0",
```

Comment:
version `5.1.0` doesn't exist for this package https://www.npmjs.com/package/@material/circular-progress